src/main/java/ay2021s1_cs2103_w16_3/finesse/model/ReadOnlyFinanceTracker.java

```diff
@@ -24,4 +25,9 @@
      * Returns an unmodifiable view of the transactions list.
      */
     ObservableList<Income> getIncomeList();
+
+    /**
+     * Returns an unmodifiable view of the transactions list.
```

Comment:
```suggestion
     * Returns an unmodifiable view of the frequent expense list.
```
Same with similar Javadocs in this class.